### PR TITLE
[package.json] Leave the protocol choice for a Github repo to npm

### LINKED ISSUE #1371: `npm install --save react-native` failed
I followed the instruction to install react-native on my macbook
http://facebook.github.io/react-native/docs/getting-started.html

It seems like to use a ssh command to fetch repository, but my ssh key .ssh/id_rsa is not used for github, so I encounter an error:
screen shot 2015-05-22 at 9 58 51 am

git fetch -a origin (git://github.com/frantic/stacktrace-parser.git)
could you turn this fetch method to https://?
thanks a lot.

### PR DESCRIPTION
**Pull request by brentvatne**
Fixes #1371

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "rebound": "^0.0.12",
     "sane": "git://github.com/tadeuzagallo/sane.git#a029f8b04a",
```
Comment (frantic) [inline package.json]: should this also be updated then?

Comment (brentvatne) [inline package.json]: Certainly

Comment (brentvatne) [inline package.json]: Fixed, thanks for catching that

**Comment by frantic** [comment]
Landed internally, should be here with the next sync. Thanks for fixing it, @brentvatne!

**Pull request closed without merging**